arborist/server.go

```diff
@@ -86,7 +86,7 @@ func (server *Server) MakeRouter(out io.Writer) http.Handler {
 	router.HandleFunc("/health", server.handleHealth).Methods("GET")
 
 	router.Handle("/auth/proxy", http.HandlerFunc(server.handleAuthProxy)).Methods("GET")
-	router.Handle("/auth/request", http.HandlerFunc(parseJSON(server.handleAuthRequest))).Methods("POST")
+	router.Handle("/auth/{identity:(?:request|client)}", http.HandlerFunc(parseJSON(server.handleAuthRequest))).Methods("POST")
```

Comment:
I would be in favor of making these completely separate endpoints/handlers on the router, and then separate the code for parsing the request etc. that can be reused between the two

Reply:
For the record, as discussed, we will keep only `/auth/request`.